provisioning/e2e/_transport_integration.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var fs = require('fs');
+var assert = require('chai').assert;
+var debug = require('debug')('azure-device-provisioning:e2e');
```

Comment:
>:e2e [](start = 55, length = 4)

nit: -e2e (name of the package). the : separator is used within the package to separate files/classes.